app/views/posts/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if user_signed_in? %>
+  <%= link_to "Sign out", destroy_user_session_path, data: { turbo_method: :delete } %>
+<% else %>
+  <%= link_to "Sign in", new_user_session_path %>
+  <%= link_to "Sign up", new_user_registration_path %>
+<% end %></p>
+
+<h1>Posts</h1>
+
+<ul>
```

Comment:
The html structure here is baffling, but I suppose that does not really matter for this exercise.